### ISSUE TITLE
"Edit profile" is not accessible

### ISSUE THREAD

**Issue by WofWca** [deltachat/deltachat-desktop#5794]
- Operating System (Linux/Mac/Windows/iOS/Android):
- Delta Chat Version:
- Expected behavior: Can open "Edit profile" dialog using just keyboard.
- Actual behavior: Can't
- Steps to reproduce the problem:
- Screenshots:
- Logs:

The bug has been introduced in https://github.com/deltachat/deltachat-desktop/pull/5765.
